test/e2e/facility-helpers.js

```diff
@@ -0,0 +1,267 @@
+const mock = require('./mock-helpers');
+
+/* eslint-disable camelcase */
+const resultsData = {
+  data: [
+    {
+      id: 'vba_343z',
+      type: 'va_facilities',
+      attributes: {
+        unique_id: '343z',
+        name: 'VetSuccess on Campus at Los Angeles City College',
+        facility_type: 'va_benefits_facility',
+        classification: 'VETSUCCESS ON CAMPUS',
+        website: null,
+        lat: 34.08751109,
+        'long': -118.2917626,
```

Comment:
Oh, got it. That's strange; they were apparently only reserved in older versions of ECMAScript (1-3) too. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Future_reserved_keywords